src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -427,6 +428,14 @@ export class PatientViewPageStore {
         invoke: async() => groupBySampleId(this.sampleIds, this.clinicalDataForSamples.result)
     }, []);
 
+    readonly clinicalDataGroupedBySampleMap = remoteData({
+        await: () => [this.clinicalDataGroupedBySample],
+        invoke: async() => {
+            let sampleIdToClinicalDataMap =  mapSampleIdToClinicalData(this.clinicalDataGroupedBySample.result, 'id', 'clinicalData');
```

Comment:
`sampleIdToClinicalDataMap ` is redundant

Reply:
@zhx828 can I get some clarification here? Do you mean I can directly return `mapSampleIdToClinicalData(...)` or do you mean `sampleIdToClinicalDataMap` is redundant to another map that is already defined somewhere?

Reply:
@averyniceday directly return it